frontend-nx/apps/edu-hub/components/pages/ManageCourseContent/ApplicationsTab/ApplicationRow.tsx

```diff
@@ -89,10 +89,10 @@ export const ApplicationRow: FC<IProps> = ({ enrollment, qResult, onSetRating, i
             <div className="mr-3 ml-3 col-span-3">{enrollment.User.lastName}</div>
             <div className="mr-3 ml-3 col-span-12 truncate">{enrollment.motivationLetter}</div>
             <div className="mr-3 ml-3 col-span-2 text-center">
-              {enrollment.motivationRating === 'UNRATED' ? greyDot : <></>}
-              {enrollment.motivationRating === 'INVITE' ? greenDot : <></>}
-              {enrollment.motivationRating === 'REVIEW' ? orangeDot : <></>}
-              {enrollment.motivationRating === 'DECLINE' ? redDot : <></>}
+              {enrollment.motivationRating === 'UNRATED' &&  <Dot color="grey" />}
```

Comment:
I think the import of the dot colors can be deleted since the Dot component is used via the string value representing the color. Or instead of `<Dot color="orange" />` one could maybe use `<orangeDot/>`